hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/HoodieParquetInputFormat.java

```diff
@@ -73,6 +73,14 @@ protected HoodieDefaultTimeline filterInstantsTimeline(HoodieDefaultTimeline tim
     return HoodieInputFormatUtils.filterInstantsTimeline(timeline);
   }
 
+  protected FileStatus[] getStatus(JobConf job) throws IOException {
```

Comment:
would we remove the method and call `listStatus` directly?

Reply:
no, we need to call ParquetInputFormat.listStatus in HoodieParquetRealtimeInputFormat.java, but we cannot access that method directly,   so we overwrite  getStatus in ParquetRealtimeInputFormat.java call call listStatus in it